app/reviews/[slug]/page.tsx

```
@@ -0,0 +1,230 @@
+
+  if (!review) {
+    notFound()
+  }
+
+  const linkUrl = review.affiliateUrl || review.productUrl
+  const url = `https://erichsimon.com/reviews/${review.slug}`
+  const structuredData = generateReviewStructuredData(review, url)
+
+  return (
+    <div className="container max-w-4xl py-6 md:py-16">
+      <script type="application/ld+json" dangerouslySetInnerHTML={{ __html: JSON.stringify(structuredData) }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)